src/client/components/SignupComponent/index.js

```diff
@@ -0,0 +1,83 @@
+require('./index.scss');
+const React = require('react');
+import SignupForm from '../SignupForm';
+
+class SignupComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'status': 'being',
+      'alert': '',
+    };
+  }
+  submitHandler(ev) {
+    ev.preventDefault();
+    let username = ev.target.elements.namedItem('username').value;
+    let fullname = ev.target.elements.namedItem('fullname').value;
+    let email = ev.target.elements.namedItem('email').value;
+    let phonenumber = ev.target.elements.namedItem('phonenumber').value;
+    let password = ev.target.elements.namedItem('password').value;
+    let obj = {
+      username: username,
+      fullname: fullname,
+      email: email,
+      phonenumber: phonenumber,
+      password: password,
+    };
+    let jsonData = JSON.stringify(obj);
+    this.sendData(jsonData);
+    
+  }
+
+  sendData(data) {
+    let xhr = new XMLHttpRequest();
+    xhr.addEventListener('readystatechange', function() {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
```

Comment:
i think we should change the state of the component to being in case of retrieved response